example-streamer-uses/README.md

```diff
@@ -1,45 +1,80 @@
+# How to run "up-linux-streamer"
+
+### Run following command from root directory of project:
+```bash
+
+     cargo run --bin up-linux-streamer -- --config up-linux-streamer/DEFAULT_CONFIG.json5  
+
+```
+
 # Examples
 
-## Running examples
+### Please note that these examples should be run in pairs. For instance, ue_service should be paired with me_client, and me_service should be paired with ue_client. Additionally, ensure that up-linux-streamer is running before executing these examples from root directory of project.
+
+## How to run rpc examples on same local machine. 
```

Comment:
Why have we now duplicated these sections into this one, without the details about "run this in another terminal", endpoint flag and _another_ one down below that does include that detail, and expected output?

Please revert this change and stick to a setup like this:

# Examples

## RPC

### Mechatronics client to high compute service

Show all the existing content, with any typo issues or ways to run fixed.

### High compute client to mechatronics service

Show all the existing content, with any typo issues or ways to run fixed.

## PubSub

### Mechatronics publisher to high compute subscriber

Mirror "Mechatronics client to high compute service" above, i.e. show how to run this as well as sample output to the user can see what's expected output

### High compute publisher to mechatronics subscriber

Show how to run this, feel free to put a disclaimer as I had in which we say we omit the details.